mica/web/close_events.py

```diff
@@ -0,0 +1,24 @@
+from kadi import events
+from Chandra.Time import DateTime
+
+# 30 minute pad for these events
+interest_pad = 30 * 60;
+interest = [events.normal_suns,
```

Comment:
This could be a list of tuples `(event_type, start_pad, stop_pad)`, then you don't need to special-case major_events below.  The asymmetric pad allows normal_sun, safe_suns to have pads of (e.g.) -1 hr to +3 days.  That represents the typical impact window for those events.